src/vidore_benchmark/retrievers/jina_v4_retriever.py

```diff
@@ -72,7 +72,7 @@ def get_scores(
         if self.vector_type == "single_vector":
             return self.score_single_vector(query_embeddings, passage_embeddings, device=self.device)
         elif self.vector_type == "multi_vector":
-            return self.score_multi_vector(query_embeddings, passage_embeddings, device=self.device)
+            return self.score_multi_vector(query_embeddings, passage_embeddings, device=self.device, batch_size=batch_size)
```

Comment:
Should we also pass this for `single_vector`?

Reply:
The score_single_vector function has no batch_size argument. I think it is not necessary to apply batching for single vector evaluation unless you have very large datasets (>1M documents) which we usually don't have